src/docs/user/references/Notification-service.adoc

```diff
@@ -78,6 +79,21 @@ The _Submitted by_ switch may be activated to be notified of events for job subm
 
 To modify a job subscription simply click on it, do your adjustments and save it.
 
+[[channels_subscription]]
+===== Channel subscriptions
+
+Like service subscriptions, channel subscriptions are unique to each user. It allows users to subscribe to channels created beforehand.
+A user that is a notification service administrator can subscribe to any channel. Otherwise either the channel is authorized
```

Comment:
Otherwise, a user can subscribe to:
 - a public channel (a channel with all groups authorization)
 - a channel whose group authorizations match one of the user groups.